run_update.py

```diff
@@ -347,6 +347,9 @@ def get_projects(organization):
                         # some values might be empty strings
                         elif type(project_value) in (str, unicode) and unicode(project_value.decode('utf8')) == u'':
                             project[project_key] = None
+                        # we want tags to be a list with no whitespace
+                        elif project_key == 'tags':
+                            project[project_key] = [tag.strip() for tag in project_value.split(',')]
```

Comment:
And maybe exclude empty values by adding `if tag.strip() is not u''` to the list comprehension?